apps/server/src/services/timerUtils.ts

```diff
@@ -88,6 +93,12 @@ export function getCurrent(state: RuntimeState): number {
  * @returns {boolean}
  */
 export function skippedOutOfEvent(state: RuntimeState, previousTime: number, skipLimit: number): boolean {
+  // eslint-disable-next-line no-unused-labels -- dev code path
+  DEV: {
+    if (state.timer.expectedFinish === null || state.timer.startedAt === null) {
```

Comment:
I'm also hitting this 
just by creating an event 1 minnut from now and then clicking roll

Reply:
@cpvalente 
how about this is it intended?

Reply:
This should no longer be called before an event is loaded. Is that not the case?

Reply:
looks to work fine after the recent force push
https://github.com/cpvalente/ontime/commit/d721d550b48e66b65ee358275776151bb6827b60